module/modmanager/manager.go

```diff
@@ -62,22 +70,19 @@ type Manager struct {
 
 // Close terminates module connections and processes.
 func (mgr *Manager) Close(ctx context.Context) error {
-	mgr.mu.Lock()
-	defer mgr.mu.Unlock()
 	var err error
 	for _, mod := range mgr.modules {
-		if mod.conn != nil {
-			err = multierr.Combine(err, mod.conn.Close())
-		}
-		if mod.process != nil {
-			err = multierr.Combine(err, mod.process.Stop())
-		}
+		err = multierr.Combine(err, mgr.remove(mod, false))
```

Comment:
`Close` should use `remove`, too. 

I don't think this `Close` will leave us with "orphaned" or un`Close`d resources. This method is only called [here](https://github.com/viamrobotics/rdk/blob/main/robot/impl/local_robot.go#L188). "Orphaned" resources really only apply when reconfiguring a robot, not closing one. And, these resources will have already been `Close`d when `Close` is called on the resource manager [a few lines before](https://github.com/viamrobotics/rdk/blob/main/robot/impl/local_robot.go#L185).